src/theme/FilterTab.ts

```diff
@@ -0,0 +1,42 @@
+import { mode } from "@chakra-ui/theme-tools"
+
+export type FilterTabVariant = "primary" | "inline"
+export type FilterTabSize = "xs" | "sm" | "md" | "lg"
+
+export const FilterTab = {
+  baseStyle: (props: any) => {
```

Comment:
Can we add type here? Eg. https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/tabs.ts#L1-L9